pwa/src/components/handlers/handlerForm.tsx

```diff
@@ -97,69 +96,39 @@ export default function HandlerForm({id, endpointId}) {
     // get the inputs and check if set other set null
     let body: {} = {
       name: event.target.name.value,
-      description: event.target.description ? event.target.description.value : null,
+      description: event.target.description
+        ? event.target.description.value : null,
       sequence: event.target.sequence.value
         ? parseInt(event.target.sequence.value)
         : null,
       endpoint: `/admin/endpoints/${endpointId}`,
-      entity: event.target.entity.value ? event.target.entity.value : null,
-      template: event.target.template.value ? event.target.template.value : null,
-      templateType: event.target.templateType.value ? event.target.templateType.value : null,
+      entity: event.target.entity.value
+        ? event.target.entity.value : null,
+      template: event.target.template.value
+        ? event.target.template.value : null,
+      templateType: event.target.templateType.value
+        ? event.target.templateType.value : null,
+      conditions,
+      skeletonIn,
+      skeletonOut,
+      translationIn,
+      translationOut,
+      mappingIn,
+      mappingOut,
     };
 
-    if (conditions.length !== 0) {
-      body["conditions"] = conditions;
-    } else {
-      body["conditions"] = [];
-    }
-
-    if (skeletonIn.length !== 0) {
-      body["skeletonIn"] = skeletonIn;
-    } else {
-      body["skeletonIn"] = [];
-    }
-
-    if (skeletonOut.length !== 0) {
-      body["skeletonOut"] = skeletonOut;
-    } else {
-      body["skeletonOut"] = [];
-    }
-
-    if (translationIn.length !== 0) {
-      body["translationIn"] = translationIn;
-    } else {
-      body["translationIn"] = [];
-    }
```

Comment:
Werkt hier alles nog zoals je zou verwachten? Dan is dit een hele mooie refactor 👍🏼